site-scrapers/LowellGeneral.js

```diff
@@ -0,0 +1,169 @@
+const sites = require("../data/sites.json");
+const https = require("https");
+const html_parser = require("node-html-parser");
+
+const siteName = "Lowell General";
+const site = sites[siteName];
+
+module.exports = async function GetAvailableAppointments(browser) {
+    console.log(`${siteName} starting.`);
+    const webData = await ScrapeWebsiteData(browser);
+    console.log(`${siteName} done.`);
+    return webData;
+};
+
+function leadZero(num) {
+    return num < 10 ? "0" + num : num;
+}
+
+function buildTimeSlotsUrl(stationId) {
+    let doctorId = stationId.split("_")[6];
+    let locationId = stationId.split("_")[7];
+    let stationNumber = stationId.split("_")[1].replace("appointments", "");
+    let now = new Date(new Date().getTime() - 86400 * 1000);
```

Comment:
comment here explaining the reasoning behind your arithmetic?

Reply:
Will do.